### PR TITLE
Rfi scan update

### PR DESCRIPTION
**Pull request by spassmoor**
Update the RFI scan script to work on the current system and include the option to have a longer scan time

### CODE REVIEW

File: observation/rfi_scan.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 import time
 
-import numpy as np
+#import numpy as np
```
Comment (ludwigschwardt) [inline observation/rfi_scan.py]: I would remove the line completely if not needed.

File: observation/rfi_scan.py [hunk 2]
```diff
@@ -19,8 +19,9 @@
 # Add experiment-specific options
 parser.add_option('-m', '--min-duration', type="float", default=None,
                   help="The The minimum time to repeat the rfi scan over (default=%default)")
-parser.remove_option('-f')
-parser.remove_option('-p')
+parser.add_option('--long', type="float", default=None,
```
Comment (ludwigschwardt) [inline observation/rfi_scan.py]: Why not change `--long` to `--scan-duration` with a default value of 180? You'll get the same behaviour but things will be clearer. To me, `--long` sounds like a boolean and not something that expects a duration.

File: observation/rfi_scan.py [hunk 3]
```diff
@@ -34,9 +33,7 @@
 el_end = 27.1  # 15.1
 scan_spacing = 5.0 # 6.0
 num_scans = 3
-scan_duration = 180.
-if opts.long is not None:
-    scan_duration = float(opts.long)
+scan_duration = float(opts.scan-duration)
```
Comment (ludwigschwardt) [inline observation/rfi_scan.py:36]: No need to cast to float -- it's already a float. Plus `scan-duration` would have needed an underscore. Rather do: `scan_duration = opts.scan_duration`.

**Review by ludwigschwardt — CHANGES_REQUESTED** [review]

File: observation/rfi_scan.py (hunk 3 above)
Comment (spassmoor) [reply observation/rfi_scan.py]: Done

**Review by ludwigschwardt — APPROVED** [review]

**Pull request merged**